src/workflows/steps.js

```diff
@@ -48,10 +48,30 @@ export function checkHasDevelopBranch( state ) {
 	} );
 }
 
-export function askPrereleaseIdentifier( state ) {
+export function setPrereleaseIdentifier( state ) {
 	const { identifier } = state;
+	const redundantIdentifierPrefixes = [
+		"defect",
+		"feature",
+		"rework"
+	];
+
+	const cleanIdentifier = target => {
```

Comment:
This could be greatly simplified using the regex version of the [`replace`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) method like the following...

```js
const cleanIdentifier = identifier =>
   identifier.replace( /^(defect|feature|rework)-/, "" );
```